Integrating with your webapp/README.md

```diff
@@ -0,0 +1,167 @@
+# Integrate Anbox Cloud in your web application
+
+The dashboard included in Anbox Cloud is great to get started but writing your own web application
+
+## 0. AMC
```

Comment:
We should definitely link here to the existing documentation to provide further details around amc